.env.example

```diff
@@ -25,4 +25,8 @@ ELASTIC_PASSWORD=elastic_password
 ELASTIC_INDEX=elastic_index
 ELASTIC_TX_INDEX=elastic_tx_index
 ELASTIC_RESULTS_PER_PAGE=10
-ELASTIC_SEARCH_TIMEOUT=25
\ No newline at end of file
+ELASTIC_SEARCH_TIMEOUT=25
+WALLET_SERVICE_USERNAME=wallet_service_user
+WALLET_SERVICE_PASSWORD=password
```

Comment:
I would avoid accessing directly wallet service's database, "surpassing" the service. Would it be feasible to implement an API on wallet-service to proxy this request to the database?

Reply:
We will be using a read replica and not connecting to the wallet service db directly, so it will not interfere with the wallet-service in any capacity.